extensions/functions_string.yaml

```diff
@@ -163,3 +165,236 @@ scalar_functions:
           - value: "fixedchar<L1>"
           - value: "varchar<L2>"
         return: "BOOLEAN"
+  -
+    name: ltrim
```

Comment:
My assumption was as-is, since that seems to be how Postgresql and DuckDB do it.  For regex, they use different functions.  I was planning on looking into those functions in another PR.